en/python_introduction/README.md

```diff
@@ -815,7 +815,7 @@ hi()
 
 Okay, our first function is ready!
 
-You may wonder why we've written the name of the function at the bottom of the file. This is because Python reads the file and executes it from top to bottom. So in order to use our function, we have to re-write it at the bottom.
+You may wonder why we've written the name of the function at the bottom of the file. When we write `def hi():` and the indented lines following, this is us writing instructions for what the `hi()` function should do. Python will read and remember these instructions, but won't run the function yet. To tell Python we want to run the function, we have to call the function with `hi()`. Python reads the file and executes it from top to bottom, so we have to define functions in the file before we call them.
```

Comment:
For simplicity, I'd only mention this one function:

```suggestion
You may wonder why we've written the name of the function at the bottom of the file. When we write `def hi():` and the indented lines following, this is us writing instructions for what the `hi()` function should do. Python will read and remember these instructions, but won't run the function yet. To tell Python we want to run the function, we have to call the function with `hi()`. Python reads the file and executes it from top to bottom, so we have to define the function in the file before we call it.
```

Readers can hopefully themselves conclude, that this goes for each other function, too. And
> [W]e have to define functions in the file before we call them.

could be misinterpreted such as that all functions have to be defined before you can call even one of them, which isn't true. One could define one, calls it, define the next one, call that, etc.

Reply:
Great points! I like this suggestion a lot, I will commit it.